### DISCUSSION TITLE
How to change the chat colour based on Permission plugin group/ranks.

### DISCUSSION THREAD

**Discussion by RobellaRose** [TownyAdvanced/Towny#4763] [Q&A] [answered]
I'm trying to change my staff's text color in chat. Using EssentialsX (but disabled chat from that plugin), LuckPerms for prefixes, and Towny Chat for chat management. Is there any way to give a group chat color permission? I tried setting the color text through the prefix, but it's not working. Have tried setting it up through EssentialsX and LuckPerms as well. Any help would be much appreciated.

**Comment by LlmDl** [comment, marked as answer]
To change the chat colour to something based on a player's rank or group in your permission plugin:

- Set the desired colour code in your rank/groups' suffix.
- In your chatconfig.yml' channel_format lines replace teh {msgcolor} with {permsuffix} or another variation of the suffix chat tag ({permgroupsuffix} permusersuffix}.)

**Reply by RobellaRose** [reply]
Thank you so much, this worked! You are a life saver!
